curl/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.0)
+
+project(CurlTest C)
+
+include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
+conan_basic_setup(TARGETS)
```

Comment:
So, believe it or not, you actually don't have to use the Conan way of linking against Curl. CMake provides a FindCURL module: https://cmake.org/cmake/help/v3.0/module/FindCURL.html

Reply:
Take a look at libxml2, it shows a better example for that